File: day006/day006-2.java
public class Main {
	
	public static void main(String[] args) {
		java.util.Scanner kb = new java.util.Scanner(System.in);
		float x, y;
		System.out.print("Bir sayı giriniz: X ");
		x = kb.nextFloat();
		System.out.print("Başka bir sayı daha giriniz: Y ");
		y = kb.nextFloat();
		System.out.println(x+y);
		System.out.println(x-y);
		System.out.println(x*y);
		System.out.println(x/y);
	}
}

File: day006/day006-2.c
#include <stdio.h>

int main() {
	float x, y;
	
	printf("Bir sayı giriniz ");
	scanf("%f", &x);
	
	printf("Bir sayı giriniz ");
	scanf("%f", &y);
	
	printf("%f \n", (x+y));
	printf("%f \n", (x-y));
	printf("%f \n", (x*y));
	printf("%f \n", (x/y));
	
	return 0;
}
